ServiceTier/Services/SocialAuthService.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using Common.Enums;
+using Common.ModelsDTO;
+using Newtonsoft.Json;
+using Timer.DAL.Timer.DAL.UnitOfWork;
+using Timer.DAL.Timer.DAL.Entities;
+using Timer.DAL.Extensions;
+
+namespace ServiceTier.Services
+{
+    public class SocialAuthService : ISocialAuthService
```

Comment:
do we need separate service for this? Easy way to answer - do we ever use it outside login controller / service? If no - makes sense to make this as methods of login service.

Reply:
No, now we didn't plan use this service outside login controller.